src/template_project/data_management/stats4schools_smoking_template.r

```diff
@@ -0,0 +1,72 @@
+# ======================================================================================
+# Functions
+# ======================================================================================
+clean_data <- function(data) {
+  clean <- data.frame(
+    gender = clean_gender(data$gender),
+    marital_status = clean_marital_status(data$marital_status),
+    smoke = clean_smoke(data$smoke),
+    smoke_numerical = convert_smoke_to_numerical(data$smoke),
```

Comment:
```suggestion
    current_smoker_numerical = convert_current_smoker_to_numerical(data$current_smoker),
```

I think that should actually be a second step because we should use the cleaned version, even if it does not make a difference here.

Maybe we can mirror the Python structure even more closely, not trying to do everything in one block?